docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search.rst

```diff
@@ -0,0 +1,46 @@
+.. _Searching Metadata from |product|:
+
+Searching Metadata from |product|
+=================================
+
+Every object stored in an object storage system is associated with a set
+of metadata. Understanding the metadata tags associated with these objects
+provides a powerful method for rapid object search and retrieval.
+
+The |product| UI provides a graphical tool for performing metadata searches, the
+syntax for which is hinted under the search bar itself, but also
+described explicitly in :ref:`Searching Metadata with |product|`.
+
+To search the metadata of files stored in clouds managed by |product|,
+
+#. Click the **Data Browser** tab.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser.png
+      :width: 100 %
+
+#. Select an account from the **Account Name** column.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser_account_select.png   
+      :width: 75 %
+
+#. Select a bucket to search. Either pick one directly from the **Bucket Name**
+   column or narrow your options using the filter field.
+
+   .. image:: ../../Graphics/xdm_ui_bucket_select.png
+      :width: 75 %
+
+#. Enter metadata search terms in the modified NoSQL format described in
```

Comment:
as described in?

Reply:
"as described in" introduces a weasel-word that doesn't add any value. We're going right to it in the original.